packages/git-commit-msg-linter/package.json

```diff
@@ -22,8 +22,6 @@
     "pub:patch": "npm version patch",
     "pub:minor": "npm version minor",
     "pub:major": "npm version major",
-    "lint": "eslint *.js",
-    "lint:fix": "eslint --fix *.js",
     "install": "node install.js",
     "uninstall": "node uninstall.js",
     "preversion": "npm test && npm run lint",
```

Comment:
`preversion` and `postversion` should delete first? The test is hard to run because the test script do not exist in this repo. Not very convenient for other contributor.

Reply:
i will try to make it easier